includes/API/FBE/Installation/Read/Response.php

```diff
@@ -84,11 +84,14 @@ public function get_catalog_id() {
 	 */
 	public function get_page_id() {
 
-		$pages = $this->get_data()->pages;
+		$pages = ! empty( $this->get_data()->pages ) ? $this->get_data()->pages : '';
```

Comment:
This line could remain `$pages = $this->get_data()->pages;` since you are checking for its emptiness once again below. But that does not matter much.

Reply:
Thanks for the feedback.

`$pages = $this->get_data()->pages; ` will throw the PHP notice if the property $pages does not exist.

However, I agree that having `$pages = ! empty( $this->get_data()->pages ) ? $this->get_data()->pages : '';` and exiting if page is an empty string is redundant. I modified the function as follows:

```
if ( empty( $pages = $this->get_data()->pages ) ) {
	return '';
}

return is_array( $pages ) ? current( $pages ) : $pages;
```